### DISCUSSION TITLE
v0.1.0-beta

### DISCUSSION THREAD

**Discussion by muchenhen** [muchenhen/MuQianQiu#1] [Q&A] [answered]
可牌库选择
可切换对手明牌
可以再来一局
对手暂时只有一种难度
无语音是否播放选项
无特殊牌机制
无重新选择相关设置
无音量设置

<hr /><em>This discussion was created from the release <a href='https://github.com/muchenhen/MuQianQiu/releases/tag/v0.1.0-beta'>v0.1.0-beta</a>.</em>

**Comment by Alec-Ray** [comment, marked as answer]
cool
